src/platform/forms-system/src/js/web-component-fields/VaFileInputField.jsx

```diff
@@ -39,7 +40,7 @@ const VaFileInputField = props => {
   const dispatch = useDispatch();
   const [file, setFile] = useState(null);
   const formData = useSelector(state => state.form.data);
-  const { formNumber } = formData;
+  const { formNumber } = props?.uiOptions;
```

Comment:
This wasn't actually getting the `formNumber` before.

Reply:
Oops, sorry bout that!